src/client/components/input-range/input-range.vue

```diff
@@ -68,11 +77,11 @@ export default {
       },
     }
   },
-  watch: {
-    val: debounce(function(value) {
-      this.$emit("updateClasses", [this.min, value[0], value[1], this.max])
+  methods: {
+    onInput: debounce(function(value) {
+      this.$emit('updateClasses', [this.min, value[0], value[1], this.max])
     }, 1000)
```

Comment:
Wondering if this could not be a bit less - faster updates? A second is quite long, or is this somehow matched to the map updates? 
A second question is whether we want to store this value in a constant somewhere, like `INPUT_UPDATE_INTERVAL` or something, since it's currently being used as a magic number in multiple places.